npm/pkg/dataplane/ipsets/ipset.go

```diff
@@ -230,6 +232,10 @@ func NewIPSet(setMetadata *IPSetMetadata) *IPSet {
 	return set
 }
 
+func (set *IPSet) GetSetMetadata() *IPSetMetadata {
+	return NewIPSetMetadata(set.unPrefixedName, set.Type)
```

Comment:
why do we want the unprefixed name for this?

Reply:
Metadata of a ipset should always be created with unprefixed name, or else we will end up using double prefixed names.